aesthetic_ants/penguin/constants.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class CollisionType(Enum):
+    PLAYER = 1
+    ENEMY = 2
```

Comment:
You could use `enum.auto()` here. (up to you though, requires changing the import)